modules/openxr/doc_classes/OpenXRInteractionProfileMetadata.xml

```diff
@@ -25,28 +25,36 @@
 			<param index="1" name="display_name" type="String" />
 			<param index="2" name="toplevel_path" type="String" />
 			<param index="3" name="openxr_path" type="String" />
-			<param index="4" name="openxr_extension_name" type="String" />
+			<param index="4" name="openxr_extension_names" type="String" />
 			<param index="5" name="action_type" type="int" enum="OpenXRAction.ActionType" />
 			<description>
-				Registers an input/output path for the given [param interaction_profile]. The profile should previously have been registered using [method register_interaction_profile]. [param display_name] is the description shown to the user. [param toplevel_path] specifies the bind path this input/output can be bound to (e.g. [code]/user/hand/left[/code] or [code]/user/hand/right[/code]). [param openxr_path] is the action input/output being registered (e.g. [code]/user/hand/left/input/aim/pose[/code]). [param openxr_extension_name] restricts this input/output to an enabled/available extension, this doesn't need to repeat the extension on the profile but relates to overlapping extension (e.g. [code]XR_EXT_palm_pose[/code] that introduces [code]…/input/palm_ext/pose[/code] input paths). [param action_type] defines the type of input or output provided by OpenXR.
+				Registers an input/output path for the given [param interaction_profile]. The profile should previously have been registered using [method register_interaction_profile]. [param display_name] is the description shown to the user. [param toplevel_path] specifies the bind path this input/output can be bound to (e.g. [code]/user/hand/left[/code] or [code]/user/hand/right[/code]). [param openxr_path] is the action input/output being registered (e.g. [code]/user/hand/left/input/aim/pose[/code]). [param openxr_extension_names] restricts this input/output to an enabled/available extension, this doesn't need to repeat the extension on the profile but relates to overlapping extension (e.g. [code]XR_EXT_palm_pose[/code] that introduces [code]…/input/palm_ext/pose[/code] input paths). [param action_type] defines the type of input or output provided by OpenXR.
+			</description>
+		</method>
+		<method name="register_path_rename">
+			<return type="void" />
+			<param index="0" name="old_name" type="String" />
+			<param index="1" name="new_name" type="String" />
+			<description>
+				Allows for renaming old input/output paths to new paths in order to load and process older action maps.
```

Comment:
Should the `*_rename` methods include a version as part of their logic?
For example to cover the case we need to do additional renames when going from OpenXR 1.1 to 1.2.

Reply:
> then during runtime if we detect if we're on an older version of OpenXR, we **rename it back to what was supported on that version**.

That's the part I'm concerned about; if we rename a name when going from 1.0 to 1.1, and then do it again for 1.1 to 1.2, do we have a map to track the renames so we can retrieve the older name when running on an older version of OpenXR?